Plugins/BenchmarkTool/BenchmarkTool+Export.swift

```diff
@@ -88,6 +88,56 @@ extension BenchmarkTool {
         }
     }
 
+    /// Writes raw data into a file.
```

Comment:
Without Foundation, which wasn't imported, there wasn't a super convenient way to transform an array of `[UInt8]` into `String`, so I made a variant of the `write` function that accepted `[UInt8]` and wrote the raw data into the file.

If there's a better/preferred way of tackling this, I'd be happy to do so.